src/features/hud/components/InventoryItems.tsx

```diff
@@ -4,35 +4,154 @@ import { OuterPanel, Panel } from "components/ui/Panel";
 import { Context } from "features/game/GameProvider";
 import { ITEM_DETAILS } from "features/game/types/images";
 import { InventoryItemName } from "features/game/types/game";
-import React, { useContext, useState } from "react";
+import { KNOWN_IDS, CATEGORY_ID, CATEGORY } from "features/game/types";
+import React, { useContext, useState, useRef } from "react";
+import { Tab } from "components/ui/Tab";
 
-export const InventoryItems: React.FC = () => {
+import close from "assets/icons/close.png";
+import arrowLeft from "assets/icons/arrow_left.png";
+import arrowRight from "assets/icons/arrow_right.png";
+import seed from "assets/crops/beetroot/seed.png";
+import crop from "assets/crops/sunflower/crop.png";
+import tool from "assets/tools/hammer.png";
+import nft from "assets/nfts/gnome.png";
+import food from "assets/crops/wheat/flour.png";
+import resource from "assets/resources/wood.png";
+interface Props {
+  onClose: () => void;
+}
+
+export const categories: CATEGORY[] = [
+  {
+    id: 1,
+    name: "Seeds",
+    img: seed
+  },
+  {
+    id: 2,
+    name: "Crops",
+    img: crop
+  },
+  {
+    id: 3,
+    name: "Tools",
+    img: tool
+  },
+  {
+    id: 4,
+    name: "NFTs",
+    img: nft
+  },
+  {
+    id: 5,
+    name: "Foods",
+    img: food
+  },
+  {
+    id: 6,
+    name: "Resources",
+    img: resource
+  },
+];
+
+export const InventoryItems: React.FC<Props> = ({ onClose }) => {
   const { gameService, selectedItem, shortcutItem } = useContext(Context);
+  const scrollDiv = useRef<any>(null);
+
+  const [tab, setTab] = useState<CATEGORY_ID | number>(1);
```

Comment:
Again, magic numbers can get quite confusing for a new engineer in the repo. The value `1` does not provide much meaning to them. They would need to spend more time reading other parts of the code just to understand this.

Consider the following:

```
const [tab, setTab] = useState<Tab>('Crops');
```